### PR TITLE
Add modals for Chmod & Chown + small refactor of CSS

### PR DESCRIPTION
**Pull request by DivergentEuropeans**
image

image

image

### CODE REVIEW

File: src/app/components/file-permissions-modal/file-permissions-modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+    this.mode = parseInt(modeStringSym);
+    this.formatPermissions();
+  }
+
+  savePermissions() {
+    let url :string = ZoweZLUX.uriBroker.unixFileUri('chmod', this.path, undefined, undefined, undefined, false, undefined, undefined, undefined, this.mode.toString(), this.recursive);
+    this.http.post(url, null)
+    .map(res=>{
+      if (res.status == 200) {
```
Comment (lchudinov) [inline src/app/components/file-permissions-modal/file-permissions-modal.component.ts:256]: If I re-open this dialog after changing permissions the dialog still shows old permissions. Please update the permission info in the UI after you saved them in the filesystem.

File: src/app/components/file-ownership-modal/file-ownership-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  constructor(
+    @Inject(MAT_DIALOG_DATA) data,
+    private http: Http,
+    private snackBar: MatSnackBar,
+  ) 
+  {
+    const node = data.event;
+    this.name = node.name;
+    this.path = node.path;
+    this.mode = node.mode;
```
Comment (lchudinov) [inline src/app/components/file-ownership-modal/file-ownership-modal.component.ts:41]: This dialog doesn't show the current owners when I open it.

**Review by lchudinov — CHANGES_REQUESTED** [review]

File: src/app/components/file-ownership-modal/file-ownership-modal.component.ts (hunk 2 above)
Comment (lchudinov) [reply src/app/components/file-ownership-modal/file-ownership-modal.component.ts:41]: image

Comment (DivergentEuropeans) [reply src/app/components/file-ownership-modal/file-ownership-modal.component.ts:41]: Yes agreed, we need to update /metadata call to show ownership info so I can pre-fill those

File: src/app/components/file-permissions-modal/file-permissions-modal.component.ts (hunk 1 above)
Comment (DivergentEuropeans) [reply src/app/components/file-permissions-modal/file-permissions-modal.component.ts:256]: Yes, need to add this

**Review by lchudinov — APPROVED** [review]

**Pull request merged**